src/main/java/com/tms/webApp/entity/User.java

```diff
@@ -0,0 +1,43 @@
+package com.tms.webApp.entity;
+
+public class User {
+  private String name;
+  private String lastName;
+  private int age;
+
+  public User() {
+    name = "Vasiliy";
```

Comment:
Constructor shouldn't contain initialization for fields that may be changed later - only smth constant (and in that case initialization it field or getter is better), so remove it